src/test/java/com/training/api/tests/ga/GuestAccountTests.java

```diff
@@ -0,0 +1,51 @@
+package com.training.api.tests.ga;
+
+import com.google.gson.Gson;
+import com.training.pojos.ga.validation.Request;
+import com.training.utilities.JsonReaderUtility;
+import com.training.utilities.RestEngine;
+import io.restassured.RestAssured;
+import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.util.HashMap;
+import java.util.Map;
+
+public class GuestAccountTests {
+    protected Map<String, String> map = new JsonReaderUtility().getMap();
+    private Map<String, Object> headerMap;
+    private io.restassured.response.Response response;
+
+    @BeforeEach
+    public void setup(){
+        RestAssured.baseURI = map.get("gaBaseUrl");
+        headerMap = new HashMap();
+        headerMap.put("AppKey","qP2wzibM0y9rLeRc3jAZQEoBMGgtVGj7");
+        headerMap.put("Content-Type","application/json");
+    }
+
```

Comment:
give description to this method

Reply:
@shrikanttechincal @dheerajsrinath no need to add description for test methods :)